docs/index.md

```diff
@@ -0,0 +1,30 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# The Hyperledger Technical Steering Committee
+
+Governing Documents:
+
+* [Antitrust Policy](./antitrust.md)
+* [Criteria for Promoted Release](./criteria-for-promoted-release.md)
+* [Code of Conduct](./code-of-conduct.md)
+* [Project Incubation Exit Criteria](./project-incubation-exit.md)
+* [Project Lifecycle](./project-lifecycle.md)
+* [Release Taxonomy](./release-taxonomy.md)
+
+Guidelines:
+
+* [Maintainers Guidelines](./MAINTAINERS-guidelines.md)
+
+2019-2020 term:
```

Comment:
Do we have the info to fill the TSC membership out to the beginning of the project?  While having all the committees ever show forever may be overkill for the main page (or not, I'm persuadable) I feel having that info at least available in commit logs would be useful.  If github markdown had collapsable panels that would solve the UI explosion of a full list.

Reply:
I'd like to do that in a follow up PR. We do have the info of who and what terms; it's a matter of doing the work

Reply:
https://ryjones.github.io/tsc/tsc-members.html Hope this is better?